docs/release-notes.txt

```diff
@@ -4,6 +4,115 @@
  Release Notes
 ###############
 
+**************
+ Version 0.17
+**************
+
+Version 0.17.0
+==============
+
+**Release Date:** September 28, 2021
+
+**New Features**
+
+-  Support the `MMDetection <https://github.com/open-mmlab/mmdetection>`__ library in Model Hub to
```

Comment:
Might we please add a link to the docs for how to use MMDetection from Determined as well: https://github.com/determined-ai/determined/blob/master/docs/model-hub/mmdetection/index.txt ?

Reply:
Done!